### ISSUE TITLE
Netbox 3.x support

### ISSUE THREAD

**Issue by hiddenman** [jdrew82/netbox-tunnels-plugin#5]
Hi,

Could you please update your plugin to support Netbox 3.x version?
I've already started this work and it mostly work: https://github.com/hiddenman/netbox-tunnels-plugin/commit/813a7283ac733baf4f60e8039744f4c005d9d8c9

However, Filters still do not work (as well as other things like Edit the record, Bulk Edit/etc - they aren't just written at all :).

**Comment by itsjohannawren** [comment]
Andrew, I'll happily test out what you have and let you know if I run into any issues. I'll even take a whack at fixes if I find any ;-)

**Comment by vishnubraj** [comment]
Hi @jdrew82 ,  Can you please update this?

**Comment by jdrew82** [comment]
This is on my to-do list but at this time I'm focusing on the Nautobot version primarily and plan on backporting it to this version.

**Comment by nomad-cyanide** [comment]
My 3.1.9 test-Netbox just crashes, when I try and add this plugin. Is that expected, as it does not appear to support version 3?

**Comment by jdrew82** [comment]
Correct, this plugin is a WIP and has not been updated to work with Netbox 3.x yet.
